### PR TITLE
Features/sc 7.5rev.150212

### PR DESCRIPTION
**Pull request by rdissertori**
Hi, I've added a new sitecore version (legacy) for sitecore 7.5 rev. 150212
and fixed some issues with some sc8 images (missing dbs)

Dearly note that I've modified Boot.ps1 in sitecore-sqldev to be more case insensitive
from
`$ldfPath = $mdfPath.Replace(".mdf", ".ldf")`
to
`$ldfPath = $mdfPath -ireplace [regex]::Escape(".mdf"), ".ldf"`
otherwise it will fail because the Core db is named Sitecore.Core.MDF (in uppercase)

I've put some example docker-compose files into the folder "docker-compose" which also creates a mongo db as this is needed. in the connection strings I'm using root:roman for mongodb authentication - feel free to modify this or leave it - all fine by me.

ps: I really like your project and don't just want to profit from it, so I try to contribute whenever there is time :+1:

### CODE REVIEW

**Comment by pbering** [comment]
Hi there! Thank you for your contribution and I'm happy that you like our work here :)

I see that you have place all the compose files under `/docker-compose/` instead of placing it under `/tests/` where all the other compose files are. Can you move into the same structure? ie. /tests/VERSION/PLATFORM/. I use them to verify the images works and I assume that the purpose of your compose files are the same?

**Comment by rdissertori** [comment]
yeah thought so **after** I already opened the pull request.
I will update the structure and push it into the pull request.

And yes the purpose of the compose folder is to test it - so it's just fine I will move it.

Also I think we should provide an example compose file for each sitecore version, so people can use working examples that they can modify to their own needs.

File: images/7.5 rev. 150212/windowsservercore-1803/sitecore/Sitecore/Website/App_Config/ConnectionStrings.config [hunk 1]
```diff
@@ -0,0 +1,14 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<connectionStrings>
+  <!-- 
+    Sitecore connection strings.
+    All database connections for Sitecore are configured here.
+  -->
+  <add name="core" connectionString="user id=sa;password=HASH-epsom-sunset-cost7!;Data Source=sql;Database=Sitecore.Core" />
+  <add name="master" connectionString="user id=sa;password=HASH-epsom-sunset-cost7!;Data Source=sql;Database=Sitecore.Master" />
+  <add name="web" connectionString="user id=sa;password=HASH-epsom-sunset-cost7!;Data Source=sql;Database=Sitecore.Web" />
+  <add name="analytics" connectionString="mongodb://root:roman@mongo/Sitecore_analytics" />
```
Comment (sshushliapin) [inline images/7.5 rev. 150212/windowsservercore-1803/sitecore/Sitecore/Website/App_Config/ConnectionStrings.config]: Hi @rdissertori, I experimented with mongo a bit. I believe there should be no `roman` as password here. Wouldn't it be better to use mongodb without authentication? (for dev purposes of course).

Comment (rdissertori) [reply images/7.5 rev. 150212/windowsservercore-1803/sitecore/Sitecore/Website/App_Config/ConnectionStrings.config]: sure, can do - I will change this in all the files (including docker compose ones).
I will update this PR

**Pull request merged**